k8s/stage-location-registry-api.yaml

```diff
@@ -31,16 +31,26 @@ spec:
                 secretKeyRef:
                   name: stage-location-registry-secret
                   key: MONGO_GCE_URI
-            - name: DB_NAME_PROD
+            - name: DB_NAME_STAGE
```

Comment:
what happened to the `DB_NAME_PROD` key? Dont we need it?

Reply:
@ja-odur there are two different yaml files for production and staging. This particular one is for staging

Reply:
I get it now. Thanks